content/attributes/group_permissions.yml

```diff
@@ -0,0 +1,13 @@
+---
+type: object
+
+description:
+  The permissions that the authenticated user has for a group.
+
+properties:
+  canInviteAsCollaborator:
```

Comment:
We don't use camelCase in V2. Please use snake_case.

Reply:
That was me....